lib/MusicBrainz/Server/Entity/WorkLanguage.pm

```diff
@@ -20,6 +20,15 @@ has language => (
     isa => 'Language',
 );
 
+sub TO_JSON {
+    my ($self) = @_;
+
+    return {
+        language_id => $self->language_id,
+        language => $self->language->TO_JSON,
+    };
```

Comment:
One potential issue with that is `WorkLanguage` has an `edits_pending` property to indicate if a particular entry is being edited. We don't appear to using it, actually, but we probably should later.

Reply:
(property from Role::Editable through Role::ValueSet)

`language.id` would still be a better fit than `language_id` as it will be needed for release editor.

Reply:
Agreed that we can remove `language_id` at least.